airbyte-integrations/connectors/source-github/source_github/schemas/assignees.json

```diff
@@ -2,8 +2,8 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
   "properties": {
-    "_ab_github_repository": {
-      "type": ["string"]
+    "repository": {
+      "type": ["null", "string"]
```

Comment:
I removed `Null` from the type enumeration, although I don't think that would cause a problem.
As far as the `organization` is concerned, I do not think that this is necessary, since we still use `repositories`, not organizations. `Organizations` are only needed to get the `repositories` and further work with them. And `full_name` of each repository consists of `owner` (which is sometimes the `organization`) and a `name` (`owner/name`).

Reply:
Ok.